specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/azureADOnlyAuthentication.json

```diff
@@ -0,0 +1,299 @@
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get a list of Azure Active Directory Only Authentication property": {
+            "$ref": "./examples/ListAzureADOnlyAuthentication.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureADOnlyAuthenticationProperties": {
+      "description": "Properties of a active directory only authentication.",
+      "required": [
+        "azureADOnlyAuthentication"
+      ],
+      "type": "object",
+      "properties": {
+        "azureADOnlyAuthentication": {
+          "description": "Azure Active Directory only Authentication enabled.",
+          "type": "boolean"
+        },
+        "state": {
+          "description": "property configuration state",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Consistent",
+            "InConsistent",
+            "Updating"
+          ],
+          "x-ms-enum": {
+            "name": "state value",
+            "modelAsString": true
+          }
+        },
+        "creationDate": {
+          "description": "property configuration date",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "AzureADOnlyAuthentication": {
+      "description": "Azure Active Directory Only Authentication Info",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "azureADOnlyAuthentication resource properties",
+          "$ref": "#/definitions/AzureADOnlyAuthenticationProperties",
+          "x-ms-client-flatten": true
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true
+        }
+      }
+    },
+    "AzureADOnlyAuthenticationListResult": {
+      "description": "A list of active directory only authentications.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureADOnlyAuthentication"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "workspaceNameParameter": {
+      "name": "workspaceName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The name of the workspace",
+      "x-ms-parameter-location": "method"
+    },
+    "azureADOnlyAuthenticationNameParameter": {
+      "type": "string",
+      "in": "path",
+      "name": "azureADOnlyAuthentication",
```

Comment:
>azureADOnlyAuthentication [](http://example.com/codeflow?start=15&length=25)

azureADOnlyAuthenticationName?

Reply:
Fixed this to be consistent with workspaceName